### ISSUE TITLE
Relatives paths getting Broken (../ being stripped from paths)

### ISSUE THREAD

**Issue by taibhse-designs** [jakubpawlowicz/clean-css#1100]
No matter what I try, the ../ gets stripped from the image path resulting in a broken link and no image being displayed.

### Configuration options
Clean CSS is part of the minify plugin / extension for Brackets, configs appear to be all defaults I think.

### Input CSS

```css
.hire-me {
    background: url("../images/hire.jpg");
}
```

### Actual output CSS

```
.hire-me {background: url(images/hire.jpg);}
```

### Expected output CSS

```
.hire-me {background: url(../images/hire.jpg);}
```

**Comment by TobbeLino** [comment]
Check out https://stackoverflow.com/questions/43786547/how-to-keep-my-relative-paths-with-clean-css-minifying
We also stumbled upon this problem, and setting "rebase" = false got the desired output.

The "rebase"-feature seems to be enabled by default, but I suspect it may be broken when running on Windows. We have a node application that has been running on Windows for some time, generating relative URLs without changing them (generating your "Expected output CSS"). But when we switched to running the same application through Docker, it started removing "../" from all relative URLs. Maybe that's what "rebase" is supposed to do, but the feature did *NOT* seem to work (or at least did not work in the same way) when running directly in Windows... 
Feels like there may be a Windows/Linux path-separator issue causing this.
